third_party/nvfuser/csrc/transform_view.cpp

```diff
@@ -502,15 +502,15 @@ class AnalyzeViewTransformation {
       prev_new_view_index = new_view_index;
       prev_original_view_index = original_view_index;
 
-      if (new_view_index >= new_view_.size()) {
+      if (new_view_index >= (int64_t)new_view_.size()) {
         TORCH_INTERNAL_ASSERT(
             current_size == 1,
             "View is complete, but there's still some elements to distribute.");
       }
 
-      if ((new_view_index + 1 >= new_view_.size() ||
-           (new_view_.at(new_view_index + 1) != 1)) &&
-          original_view_index + 1 < original_view_.size() &&
+      if ((new_view_index + 1 >= (int64_t)new_view_.size() ||
+           (new_view_[new_view_index + 1] != 1)) &&
```

Comment:
Are you sure you don't want `at()` here for safety like the original?

Reply:
Good catch, will update!